cg/cli/upload/genotype.py

```diff
@@ -37,7 +37,7 @@ def upload_genotypes(context: CGConfig, re_upload: bool, family_id: Optional[str
     if not family_id:
```

Comment:
```suggestion
    if not case_id:
```
Are we catching these instances in this PR as well or out of scope?